webapp/e2e/cypress/fixtures/users.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+interface User {
```

Comment:
It seems weird to re-define this here. Should we import from redux instead?

Reply:
This was removed. The webapp is now imported https://github.com/mattermost/mattermost-plugin-demo/pull/130/commits/01dcd08e16cab3dfe0a9040c4467d8825ae30586